apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/PoolRewrite/SessionPoolOptions.cs

```diff
@@ -0,0 +1,233 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
nit: As @chrisdunelm would say, this is so last year :)

Reply:
This is *effectively( not a new file though - we don't change copyright on existing files. While *at the moment* it's a new file, it will end up being the replacement content for an existing file. I'll leave this as it is, but it's definitely an odd case until we move it back in.

Reply:
Ah OK!